libs/icons/readme.md

```diff
@@ -0,0 +1,48 @@
+# Sage Icons
+
+This is the Icon Library for Sage
+
+
+## Figma Export
+In order to export icons you will need to follow the steps listed below.
+
+### Creating a Figma access token
+1. Navigate to [Figma](https://www.figma.com) and login with your credentials.
+2. Click you Profile avatar and select 'Settings'
+3. Scroll down until you see 'Personal access tokens'
+4. Under 'Create a new personal access token:' enter a token description and hit enter
```

Comment:
What would be a good example to write here?

Reply:
I used `Kajabi`, but it literally could have been anything.